dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -26,6 +26,7 @@
 //= require md5
 
 // Application assets
+//= require namespace
```

Comment:
not sure... also that code is still in the file itself, is that intentional? (it's not going to hurt anything of course since it's a ||=)